internal/core/command/put.go

```diff
@@ -44,5 +50,7 @@ func NewPutCommand(
 		request.Header.Set(clients.CorrelationHeader, correlationID.(string))
 	}
 
+	request.Header.Set(clients.ContentType, r.Header.Get(clients.ContentType))
```

Comment:
Were you able to verify if the `Content-Length` header was being added to the request by the underlying HTTP client?

Reply:
✅ Yes, I verified that the Content-Length part of the header was being added to the request by the underlying HTTP client. Here, for handy reference, is the `golang` reference online for [Package http](https://golang.org/pkg/net/http/):
```
    // For client requests, certain headers such as Content-Length
    // and Connection are automatically written when needed and
    // values in Header may be ignored. See the documentation
    // for the Request.Write method.
```